### PR TITLE
Replace the use of glBlitFramebuffer in glwindow by a shader

### PR DESCRIPTION
**Pull request by asajeffrey**
This doesn't change much by itself, but means we can use more interesting shaders later on.

### CODE REVIEW

File: webxr/glwindow/mod.rs [hunk 1]
```diff
@@ -380,3 +345,129 @@ impl GlWindowDevice {
...
+";
+
+const FRAGMENT_SHADER: &[u8] = b"
+  #version 330 core
+  layout(location=0) out vec4 color;
+  uniform sampler2D image;
+  void main() {
+    ivec2 size = textureSize(image, 0);
+    vec2 position = vec2(gl_FragCoord.x / size.x, gl_FragCoord.y / size.y);
+    color = texture(image, position);
```
Comment (pcwalton) [inline webxr/glwindow/mod.rs:375]: This is fine, but I just use

    // vertex shader
    out vec2 vTexCoord;
    ...
    void main() {
        gl_Position = vec4(coord, 0.0, 1.0);
        vTexCoord = coord * 0.5 + 0.5;
    }

    // fragment shader
    ...
    in vec2 vTexCoord;
    void main() {
        color = texture(image, vTexCoord);
    }

Comment (asajeffrey) [reply webxr/glwindow/mod.rs]: Done.

**Comment by asajeffrey** [comment]
@Manishearth r?

**Pull request merged**